sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -607,6 +607,10 @@ def n_iter_(self):
         return len(self._predictors)
 
     def _more_tags(self):
+        # This is not strictly True, but it's needed since
+        # force_all_finite=False means accept both nans and infinite values.
+        # Without the tag, common checks would fail.
+        # This comment must be removed once we merge PR 13911
```

Comment:
Maybe add a "TODO", we sometimes go through them and it'll be easier to find it then. But if you're gonna fix it yourself, then no big deal.